src/Management/src/OpenCensus.Exporter.Zipkin/OpenCensus.Exporter.Zipkin.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) != 'Windows_NT'">netstandard2.0</TargetFrameworks>
+    <IncludeSymbols>True</IncludeSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup>
```

Comment:
I think we need to set a PackageId
```suggestion
  <PropertyGroup>
    <PackageId>Steeltoe.Management.OpenCensus.ZipkinExporter</PackageId>
```

Reply:
Not really sure what the right name is?

Reply:
I didn't see this comment about name earlier ...